common/src/main/resources/messages/stack-status-messages.yaml

```diff
@@ -0,0 +1 @@
+'cluster-proxy.ccm.endpoint-unavailable': 'The Control Plane was not able to establish the connection with the CM server instance. This means that the reverse SSH tunnel (autossh process) running on the instance could not connect to the Cloudera server. Please check your connection and proxy settings and make sure the instance can reach *.ccm.cdp.cloudera.com'
```

Comment:
I think it can be misleading to call it `CM server instance`, especially this is used for FreeIPA only

Reply:
Good point.